pygmt/src/grdlandmask.py

```diff
@@ -45,6 +49,41 @@ def grdlandmask(**kwargs):
         in.
     {I}
     {R}
+    {A}
+    resolution : str
+        *res*\[\ **+f**\]. Selects the resolution of the data set to use      **f**\|\ **h**\|\ **i**\|\ **l**\|\ **c**
```

Comment:
Not sure what happened here...
```suggestion
        *res*\[\ **+f**\]. Selects the resolution of the data set to use
```